lambdas/services/post_fhir_document_reference_service.py

```diff
@@ -62,18 +44,22 @@ def process_fhir_document_reference(
             )
 
             # Extract NHS number from the FHIR document
-            nhs_number = self._extract_nhs_number_from_fhir(validated_fhir_doc)
-            patient_details = self._check_nhs_number_with_pds(nhs_number)
+            try:
```

Comment:
Do we have a test for these new try/excepts? I couldn't see any but maybe I missed it.